requests/validate_request.go

```diff
@@ -34,7 +34,14 @@ func ValidateRequestSchema(
 	schema *base.Schema,
 	renderedSchema,
 	jsonSchema []byte,
+	opts ...Option,
 ) (bool, []*errors.ValidationError) {
+
+	config := configOptions{}
+	for _, opt := range opts {
+		opt(&config)
+	}
```

Comment:
Here too

I don't think you need a config in the struct, and a configOptions struct